MOBILE_CLIENT/IOS/_SECURE/IPA_SECURE_PRIVACY_MANIFEST_FILE/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+  "risk_rating": "secure",
+  "short_description": "The application and its third-party SDK declares Privacy manifest files.",
+  "references": {
+        "Privacy manifest files": "https://developer.apple.com/documentation/bundleresources/privacy_manifest_files",
+        "Describing use of required reason API": "https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api"
+  },
+  "title": "Privacy manifest files",
+  "privacy_issue": true,
+  "security_issue": false,
+  "categories": {
+    "OWASP_MASVS_L1": [
+      "MSTG_ARCH_12"
+    ],
+    "OWASP_MASVS_L2": [
+      "MSTG_ARCH_12"
+    ],
+    "GDPR": [
+      "ART_5",
+      "ART_25",
+      "ART_32",
+      "ART_35"
+    ],
+    "PCI_STANDARDS":[
+      "REQ_2_2",
+      "REQ_6_2",
+      "REQ_6_3",
+      "REQ_7_3"
+    ]
+  }
+}
```

Comment:
Don't we usually say: "This entry is informative, no recommendations applicable." instead of an empty recommendation file?

Reply:
+1